src/components/confirmationPopup.js

```diff
@@ -0,0 +1,28 @@
+import Popup from "./Popup";
```

Comment:
`this._handleEscapeClose` doesn't exist in `Popup`. Please, correct the `Escape` listener  removing